README.md

```diff
@@ -159,6 +159,14 @@ Identity type, for example, could provide `Id` as its type representative:
 If a type provides a type representative, each member of the type must have
 a `constructor` property which is a reference to the type representative.
 
+
+## `forAll`
+
+[Parametric polymorphism][pp] is essential to these algebras.  When the rules for any type are specified as applied to all values (`forall a b`) of the given generic types (here `a` and `b`), the rule must hold for any Javascript values unless otherwise specified.  This implies, for instance, that code that behaves differently for the `null` or `undefined` value than for other values is unlikely to comply with the rule.
```

Comment:
Could you wrap this paragraph at 79 characters?

Reply:
Please change “Javascript” to “JavaScript” for consistency with the rest of the document.

Reply:
Done and done.